index.php

```diff
@@ -25,15 +38,18 @@
             </a>
 
             <div class="dropdown-menu scrollable-menu" id="dropdown-content" aria-labelledby="dropdownMenuLink">
-                <?php $breed_id = 31; ?>
-                <!--links to populate with foreach loop in php-->
-                <a class="dropdown-item" href="http://192.168.20.20/2019-rmp-doginator/gallery.php?id=<?php echo $breed_id; ?>">Action</a>
-
+                <?php
+                foreach ($allDogEntities as $dogEntity) {
+                    $id = $dogEntity->getBreedId();
+                    $breed = $dogEntity->getBreed();
+                    echo '<a class="dropdown-item" href="http://192.168.20.20/2019-rmp-doginator/gallery.php?id=' . $id . '"' . '>' . $breed . '</a>';
```

Comment:
didn't need to concatenate " and > separately :) 
use <a> tag this instead:
`'<a class="dropdown-item" href="http://192.168.20.20/2019-rmp-doginator/gallery.php?id=' . $id . '">' . $breed . '</a>'`